src/friendly-iframe-embed.js

```diff
@@ -368,10 +369,17 @@ export class FriendlyIframeEmbed {
       ? this.host.signals()
       : new Signals();
 
+    /** @private @const {!Deferred} */
+    this.renderComplete_ = new Deferred();
+
     /** @private @const {!Promise} */
     this.winLoadedPromise_ = Promise.all([loadedPromise, this.whenReady()]);
     if (this.ampdoc) {
-      this.whenReady().then(() => this.ampdoc.setReady());
+      // TODO(ccordry): wait for renderComplete after no signing launch.
+      const readyPromise = this.spec.skipHtmlMerge
```

Comment:
It's probably best to just use one promise here: `whenRenderComplete()`, but resolve `renderComplete_` promise at the same time as `readyPromise`  when `this.spec.skipHtmlMerge`. That way this new promise will always be "correct" from API point of view. Otherwise, right now, it could end up unresolved in some cases.

Reply:
Done.